docs/source/configuration/authorization.mdx

```diff
@@ -639,12 +639,17 @@ authorization:
 
 This option will entirely reject queries that would have some parts filtered by authorization directives. The response will contain the list of paths that are affected.
 
-### log_errors
+### errors
 
 ```yaml title="router.yaml"
 authorization:
   preview_directives:
-    log_errors: true
+    errors:
+      log: true # default: true
+      response: "errors" # possible values: "errors" (default), "extensions", "disabled"
 ```
 
-By default, when part of a query is filtered by authorization, the list of filtered paths is added to the response and logged by the router. Filtering parts of a query according to the client's rights may be seen as normal operation, and not warrant investigation by platform operators, so those logs can be disabled.
\ No newline at end of file
+By default, when part of a query is filtered by authorization, the list of filtered paths is added to the response and logged by the router. We may need more flexibility in reporting errors, for various reasons:
+- filtering parts of a query according to the client's rights may be seen as normal operation, and not warrant investigation by platform operators, so logs should be disabled
+- when testing authorization, we may not want the clients to raise exceptions if authorization errors are present in the response's `errors` field, so we can put them in `extensions` instead
+- we may not want to give the clients any information about authorization errors, so they can be disabled entirely
```

Comment:
Suggestion that doesn't render nicely as a suggest:

By default, when part of a query is filtered by authorization, the list of filtered paths is added to the response and logged by the router. This is behavior can be customized for your needs.

#### `log`
By setting the enabled option you can choose if query filtering will result in a log event being output.
  
```yaml title="router.yaml"
authorization:
  preview_directives:
    log_errors: true
    errors:
      log: false # default: true
```

If filtering parts of a query according to the client's rights may be seen as normal operation, and not warrant investigation by platform operators, logging should be disabled.

#### `response`

By default filtered are places in the errors part of the graphql response.

You can configure `response` to be:
* `errors` (default) - to place filtered paths in graphql errors.
* `extensions` - to place filtered paths in extensions. Useful to suppress exceptions on the client side while still giving information that parts of the query were filtered.
* `disabled` - to suppress all information that the query was filtered.


```yaml title="router.yaml"
authorization:
  preview_directives:
    log_errors: true
    errors:
      response: errors # errors (default) | extensions | disabled
```